doc/release/3.0.0.rst

```diff
@@ -316,13 +316,15 @@ To enable the flight recorder in a YAML file, set ``flightrec.enabled`` to true:
 
 Then, you can use the Lua API to open and read ``*.ttfr`` files:
 
+.. _3-0-flight-recorder-begin:
+
 ..  code-block:: tarantoolsession
 
     app:instance001> flightrec = require('flightrec')
     ---
     ...
 
-    app:instance001> flightrec_file = flightrec.open('var/lib/instance001/20231225T085435.ttfr')
+    app:instance001> flightrec_file = flightrec.open('var/lib/instance001/current.ttfr')
```

Comment:
Don't quite understand the reason of this change. After the crash, the flight recorder rotates the log and save it in a file with a timestamp. So reading a file after the crash looks more logical IMO.